torchgeo/datasets/ssl4eo_benchmark.py

```diff
@@ -0,0 +1,369 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""Self-Supervised Learning for Earth Observation Landsat Benchmark Evaluation."""
```

Comment:
```suggestion
"""Self-Supervised Learning for Earth Observation Benchmark Datasets."""
```
No longer specific to Landsat with the more general naming scheme. Could add other satellites in the future